src/index.html

```diff
@@ -6,123 +6,123 @@
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Lora:ital,wght@1,500&family=Open+Sans:wght@300;400&display=swap" rel="stylesheet">
-  <title>What's Cookin'</title>
+  <title>what's cookin'?</title>
 </head>
 <body>
   <nav>
     <div class="nav-div flex-row">
-      <h1>What's Cookin'</h1>
+      <h1>what's cookin'?</h1>
       <input class="input" type="text" id ="searchInput" placeholder="Search recipes...">
-      <button id ="searchRecipes">Search</button>
+      <button id ="searchRecipes">search</button>
     </div>
     <div class="nav-div flex-row">
-      <button class="hidden">Home</button>
-      <button>Recipes to Cook</button>
-      <button>Favorite Recipes</button>
+      <button class="hidden">home</button>
+      <button>cookbook</button>
+      <button>favorite recipes</button>
     </div>
   </nav>
   <main>
     <div class="flex-row main-view-container main-view-container-js">
       <section class="filter-section filter-section-js">
-        <h2>Filter</h2>
+        <h3>imma lookin'</h3>
         <div class="filters">
           <div class="single-filter">
             <label for="antipasti">
-              <input class="filter-checkbox tag" type="checkbox" id="antipasti" value="yes">Antipasti
+              <input class="filter-checkbox tag" type="checkbox" id="antipasti" value="yes">antipasti
```

Comment:
curious to see what the "yes" value does! so much info in one small input element